src/components/templates/Auditorium/components/SeatingBlock/SeatingBlock.tsx

```diff
@@ -44,7 +44,7 @@ export const SeatingBlock: React.FC<SeatingBlockProps> = ({ space }) => {
 
   return (
     <Switch>
-      <Section venue={space} sectionId={sectionId} user={user} />
```

Comment:
We only allow for one section at the moment, this change isn't necessary

Reply:
What change?

Reply:
Regardless what other places of the code and the DB have, according to the line above:

`const sectionId = urlSectionId || sortedSections[0]?.id;` 

And according to the fact I did have page breaking error without the check above and the one on this line, I think it's better safe than sorry.